idaes/apps/grid_integration/forecaster.py

```diff
@@ -0,0 +1,27 @@
+import pandas as pd
+
+class WhiteNoiseForecaster:
```

Comment:
Needs doc string. Does this read in forecasts from file or add white noise? If it just reads from file, rename the class accordingly.